src/components/MultiEmailOverride.scss

```diff
@@ -1,8 +1,8 @@
 @import '~react-multi-email/style.css';
 
 .react-multi-email [data-tag] {
-  color: white;
-  background-color: #193163;
+  color: black;
+  background-color: #14306724;
```

Comment:
Pretty sure this isn't a valid hex color, right?

Reply:
Its valid. The last two hex-digits specify opacity.

<img width="513" alt="screenshot 2018-11-03 at 2 44 04 am" src="https://user-images.githubusercontent.com/23443586/47934593-a1b1f580-df12-11e8-9e2e-0487b433c559.png">

Reply:
Ah TIL. Just note that many browsers (which a surprising number of people use, based on our NUSMods analytics) don't support this, most notably Edge. https://caniuse.com/#feat=css-rrggbbaa